docs/pages/docs/migration-guide.mdx

```diff
@@ -7,6 +7,83 @@ import { FileTree, Steps, Tabs, Callout } from "nextra/components";
 
 # Migration guide
 
+## 0.9.0-next.0
+
+### API functions
+
+API functions are required and Ponder will fail to start if `src/api/index.ts` does not exist.
+
+Copy the following code into `src/api/index.ts` to achieve the same functionality as `<=0.8`.
+
+```ts filename="src/api/index.ts"
+import { db } from "ponder:api";
+import schema from "ponder:schema";
+import { Hono } from "hono";
+import { graphql } from "ponder";
+
+const app = new Hono();
+
+app.use("/graphql", graphql({ db, schema }));
+
+export default app;
+```
+
+#### Removed `ponder.get()`
+
+The `ponder.get()` function was removed. Ponder now requires the explicit use of **Hono with a default export**.
+
+<div className="code-columns">
+
+```ts filename="src/api/index.ts (<=0.8)"
+import { ponder } from "ponder:registry";
+
+ponder.get("/hello", (c) => {
+  return c.text("Hello, world!");
+});
+```
+
+```ts filename="src/api/index.ts (0.9)"
+import { Hono } from "hono";
+
+const app = new Hono();
+
+app.get("/hello", (c) => {
+  return c.text("Hello, world!");
+});
+
+export default app;
+```
+
+</div>
+
+#### `import { db } from "ponder:api"`
+
+The `db` object is now injected into the API function context. `c.db` is removed.
```

Comment:
The `c.db` object was removed from the Hono context. Now, use the `"ponder:api"` virtual module to access a readonly Drizzle database object.